keps/sig-api-machinery/2896-openapi-v3/README.md

```diff
@@ -336,6 +340,10 @@ For alpha, unit tests will be included to ensure that v3 schemas are properly ge
 - Feature implemented behind a feature flag
 - Initial e2e tests completed and enabled
 
+#### Beta
+
+- Native types are updated to capture capabilities introduced with v3
+
```

Comment:
- fix the definition names of native resources (we don't want to leak package names people start depending on)